src/main/java/uk/gov/hmcts/sptribs/citizen/event/CicSubmitCaseEvent.java

```diff
@@ -212,35 +213,36 @@ private CaseData getCaseData(final CaseData caseData, final DssCaseData dssCaseD
         List<ListValue<DssMessage>> listValues = new ArrayList<>();
 
         if (isNotEmpty(dssCaseData.getOtherInfoDocuments())) {
-            //For showing additional information page data on tab
             for (ListValue<EdgeCaseDocument> documentListValue : dssCaseData.getOtherInfoDocuments()) {
                 Document doc = documentListValue.getValue().getDocumentLink();
+                String documentComment = documentListValue.getValue().getComment();
                 doc.setCategoryId(DocumentType.DSS_OTHER.getCategory());
                 CaseworkerCICDocument caseworkerCICDocument = CaseworkerCICDocument.builder()
                     .documentLink(doc)
                     .documentCategory(DocumentType.DSS_OTHER)
+                    .documentEmailContent(documentComment)
                     .build();
 
                 if (!docList.contains(caseworkerCICDocument)) {
                     docList.add(caseworkerCICDocument);
                 }
 
-                final User caseworkerUser = idamService.retrieveUser(request.getHeader(AUTHORIZATION));
-
-                final DssMessage message = DssMessage.builder()
```

Comment:
We still need to add the additional info to the messages tab. Please use CicDssUpdateCaseEvent as an example since you are trying to replicate the behaviour in Update Case